fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/ResourcesConfig.java

```diff
@@ -62,6 +65,10 @@
   @Setter
   private String clouddriverEndpoint;
 
+  @Value("${services.igor.baseUrl}")
```

Comment:
This needs to be `${services.igor.base-url}` now.

Reply:
Done!